### ISSUE TITLE
NVDA doesn't announce keyboard layout changes (Win+Space / Alt+Shift) in the WhatsApp app

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#19751]
### Brief summary

NVDA doesn't announce the new keyboard layout when switching between layouts by pressing windows+space on whatsapp

### Steps to reproduce

1. Open whatsapp app, the new whatsapp web version.
2. Press windwos+space to switch between keyboard layouts
3. NVDA won't announce the new switeched layout.

### Actual behavior

NVDA doesn't announce the new keyboard layout

### Expected behavior

NVDA should announce the new layout, just like in any other app

### NVDA type

installed copy

### NVDA version

/

### Windows version

Windows 11

### Name and version of other software in use when reproducing the issue

Whatsapp windows store app

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by zstanecic** [comment]
Hi @fatihyuksek1 
The same is when the keystroke alt+shift is pressed. I confirm this.

**Comment by gerald-hartig** [comment]
Welcome @fatihyuksek1,

We are unable to process this issue as it stands.

- Please fill out the issue template fully and include the versions of NVDA & whatsapp, and the result when add-ons are disabled and the result of applying the COM registration tool.
- Please reproduce this issue with the above conditions and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

**Comment by fatihyuksek1** [comment]
Hello,
When you open the latest version of Whatsapp from windows store and press windows+space or alt+shift, does NVDA announce the new keyboard layout when multiple keyboard layouts are installed, for example English, Spanish and German?
That is the question and 1 other person has already confirmed my issue. It's quite simple actually to try it out and find out whether the bug is present yourself.
Thank you.
________________________________
От: gerald-hartig ***@***.***>
Отправлено: 10 марта 2026 г. 2:30
Кому: nvaccess/nvda ***@***.***>
Копия: Fatih Yüksek ***@***.***>; Mention ***@***.***>
Тема: Re: [nvaccess/nvda] NVDA doesn't announce keyboard layout changes (Win+Space / Alt+Shift) in the WhatsApp app (Issue #19751)

[https://avatars.githubusercontent.com/u/153188145?s=20&v=4]gerald-hartig left a comment (nvaccess/nvda#19751)<https://github.com/nvaccess/nvda/issues/19751#issuecomment-4027597646>

Welcome @fatihyuksek1<https://github.com/fatihyuksek1>,

We are unable to process this issue as it stands.

  *   Please fill out the issue template fully and include the versions of NVDA & whatsapp, and the result when add-ons are disabled and the result of applying the COM registration tool.
  *   Please reproduce this issue with the above conditions and provide a log file<https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps> of the behaviour. Ensure your log level is set to debug in general preferences.

Kind Regards,
NV Access Software Developers

―

**Comment by seanbudd** [comment]
Hi @fatihyuksek1, please fill out the issue template as requested or we will have to close the issue.
